lib/src/main/java/com/thenewboston/utils/PaginationOptions.kt

```diff
@@ -0,0 +1,25 @@
+package com.thenewboston.utils
+
+data class PaginationOptions(
+    val offset: Int,
+    val limit: Int
+) {
+    fun toQuery() = "?offset=$offset&limit=$limit"
+}
+
+object PAGE {
```

Comment:
What's the reason for having this object? The API will return a default page if we don't modify the query ourselves.